python/sidecar/http.py

```diff
@@ -0,0 +1,85 @@
+from fastapi import FastAPI
+
+import sidecar
+from sidecar.typing import (
+    ChatRequest,
+    ChatResponse,
+    DeleteRequest,
+    DeleteStatusResponse,
+    IngestRequest,
+    IngestResponse,
+    IngestStatusResponse,
+    ReferenceUpdate,
+    RewriteRequest,
+    RewriteResponse,
+    SearchRequest,
+    SearchResponse,
+    TextCompletionRequest,
+    TextCompletionResponse,
+    UpdateStatusResponse,
+)
+
+app = FastAPI()
+v0 = FastAPI()
```

Comment:
`v0` -> `api`?

Reply:
do you think these endpoints should live under `/api/` or `/api/v0`? I'm indifferent.

if the latter, if feels like it makes sense to scope them to `v0` or `api_v0`